installer/openlane.sky130a/construct.yaml

```diff
@@ -0,0 +1,15 @@
+name: openlane-sky130a
+version: 0
+
+channels:
+  - https://conda.anaconda.org/litex-hub
```

Comment:
`litex-hub` alone should also work, isn't it?

Reply:
Since all the constructor example were using fully qualified URLs: https://github.com/conda/constructor/search?q=conda.anaconda.org I assumed it was best-practice.

Reply:
It seems channel names are enough for other channels than main indeed: https://github.com/restlessronin/lila-deep/blob/415fc7172c49e0a1ae766c7c9331d0d85645ef33/.colab/construct.yaml

Though I found no `construct.yaml` with plain `main` on GH (https://github.com/search?q=main+extension%3Ayaml+filename%3Aconstruct.yaml&type=Code) so I have no preference on full URLs vs channel names.